crane/container_test.go

```diff
@@ -299,3 +300,11 @@ func TestOptBoolYAML(t *testing.T) {
 	err := yaml.Unmarshal([]byte("OptBool: notaboolean"), &wrapper)
 	assert.Error(t, err)
 }
+
+func TestBuildArgs(t *testing.T) {
+	var c *container
+	// Simple case
```

Comment:
Sorry if I was unclear - the problem is not the test itself, it's the comment here that doesn't add much value because there is no other scenario, and "Simple" doesn't say much about the scenario, therefore I think it's not worth keeping